terraform/variables.go

```diff
@@ -88,10 +109,46 @@ func mergeInputs(inputsByFile InputsByFile) ValueMap {
 
 	for _, fileName := range prioritisedFileNames {
 		inputsMap := inputsByFile[fileName]
+
 		for input, value := range inputsMap {
 			combinedInputs[input] = value
 		}
 	}
 
 	return combinedInputs
 }
+
+var maxLocalsDerefIterations = 32
+
+func dereferenceLocals(localExprsMap ExpressionMap, inputs ValueMap) ValueMap {
```

Comment:
when I tried to read this word, I almost choked 😄 why not add a few more characters and fill it out?
```suggestion
func dereferenceLocals(localExpressionMap ExpressionMap, inputs ValueMap) ValueMap {
```

Reply:
This change was part of the locals that reference other local commit so we will follow up on this review in the other PR.